docs/tutorial_start.md

```diff
@@ -66,30 +66,58 @@ To create our `<contact-form>` custom element, we'll use a generator to help us
 mix live_state.gen.element ContactForm contact-form
 ```
 
-This will generate an element for us in `app/js/contact-form.ts`. While LiveState itself is not tied to any specific library, for the purposes of convenience we generate an element based on the [lit](https://lit.dev) library. The generator will also install the necessary npms for you.
+This will generate an element for us in `assets/js/contact-form.ts`. While LiveState itself is not tied to any specific library, for the purposes of convenience we generate an element based on the [lit](https://lit.dev) library. The generator will also install the necessary npms for you.
 
-We'll also want to add an import for our element in `app/js/custom_elements.js`. You'll want to create this file if it doesn't exit. Add this line for the import:
+We'll also want to add an import for our element in `assets/js/custom_elements.js`. You'll want to create this file if it doesn't exit. Add this line for the import:
 
 ```javascript
+// assets/js/custom_elements.js
```

Comment:
Just curious about this commented out line

Reply:
It's redundant you're right